doc/nrf/samples/fast_pair.rst

```diff
@@ -1,7 +1,7 @@
-.. _fast_pair_samples:
+.. _bt_fast_pair_samples:
 
-Fast Pair samples
-#################
+Bluetooth Fast Pair samples
+###########################
```

Comment:
Can we move the "samples/fast_pair" path after "samples/bl" in the "nrf\doc\nrf\sample" to follow the alphabetical order?

Reply:
Yes, sorry, missed that one